scripts/convert_fortran_source.py

```diff
@@ -0,0 +1,44 @@
+import re
```

Comment:
Ah! That explains :smile: I think `YMMSL_Settings` is missing from this script!

Reply:
Updated!